embrace-android-retrofit/README.md

```diff
@@ -0,0 +1,3 @@
+# Embrace.io - Retrofit Networking Android SDK
+
+This optional add-on SDK exposes some additional networking related APIs that extends the functionality of the core Embrace.io Android SDK.
```

Comment:
This is an "add-on module" rather an an SDK? And can we be more specific about what it does, e.g. provide components to record traces around Retrofit operations?